extension/app/ts/background/settings.ts

```diff
@@ -187,3 +187,15 @@ export async function updateSimulationResults(simulationResults: SimulationResul
 		return await browser.storage.local.set({ simulationResults: SimulationResults.serialize(simulationResults) })
 	})
 }
+
+export async function saveSignerName(signerName: SignerName) {
+	return await browser.storage.local.set({ signerName: signerName})
+}
```

Comment:
This should be updated to use the new `browserStorageLocalSet` function you added in the other PR.